src/Observers/ModelObserver.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rapidez\Statamic\Observers;
+
+use Illuminate\Support\Facades\Cache;
+use Rapidez\Statamic\Models\ProductAttribute;
+use Rapidez\Statamic\Models\ProductAttributeOption;
+
+class ModelObserver
```

Comment:
This name implicates it observes all models. I think "CacheObserver" or something is a more describing name